lib/storage: enhance TSDB status response (#8721)
VictoriaMetrics/VictoriaMetrics: docs/victoriametrics/README.md

VictoriaMetrics provides UI on top of `/api/v1/status/tsdb` - see [cardinality explorer docs](#cardinality-explorer).

> VictoriaMetrics enhances Prometheus stats with `requestsCount` and `lastRequestTimestamp` for `seriesCountByMetricName`. This stats added if [tracking metric names stats](https://docs.victoriametrics.com/#track-ingested-metrics-usage) is configured.

Reviewer: ```suggestion
If [tracking metric names stats](https://docs.victoriametrics.com/#track-ingested-metrics-usage) is enabled, VictoriaMetrics enhances Prometheus object `seriesCountByMetricName` with `requestsCount` and `lastRequestTimestamp` stats.
```

Reviewer: When stats aren't enabled, it seems like client will still get `requestsCount` property in response but 0 value? If yes, how could client determine whether is actually was used 0 times or just stats are disabled?

Author: Yes, as a default value it's `0`. Client could check metric names stats page for it or this information could be added somewhere at UI.

 There is an additional issue, `TSDB status` response usually contains "daily" view of metrics, meanwhile requestsCount and lastRequestTimestamp is global.
